ui/src/api.ts

```diff
@@ -123,3 +117,36 @@ async function doGet<T>(get: Promise<AxiosResponse<T>>) {
   }
   return null;
 }
+
+/** 
+ * helper function for slicing arrays.
+ */
+function sliceArray(array: string[], size: number) {
+  var result = [];
+
+  for (var i = 0; i < array.length; i += size) {
+    result.push(array.slice(i, i+size));
```

Comment:
TODO for @andrewhead: get Jocelyn the style settings for the project

Reply:
@jocelynzsun okay, I have added style files for the project :-D Can you download this file, and then place it at `.vscode/settings.json` in the `ui/` folder? https://github.com/allenai/scholar-reader/blob/master/ui/.vscode/settings.json

Reply:
Then I would recommend checking the box for "Format on Save" in the VSCode editor settings, and then saving this file so that it gets formatted to iron out small differences in code formatting :-)